packages/react-liveness/src/components/FaceLivenessDetector/service/utils/streamProvider.ts

```diff
@@ -28,7 +28,7 @@ export interface Credentials {
   sessionToken: string;
 }
 
-const ENDPOINT = process.env.NEXT_PUBLIC_STREAMING_API_URL;
+const ENDPOINT = process && process.env.NEXT_PUBLIC_STREAMING_API_URL;
```

Comment:
`NEXT_PUBLIC_STREAMING_API_URL` is NextJS specific right? Obviously this has been in place for some time but would the usage of `NEXT_PUBLIC_STREAMING_API_URL` here  cause issues for non-NextJS consumers?

Reply:
Yeah that could be the case for sure. Further context here is this was just left in the code to provide a backdoor way for changing the streaming endpoint to be the liveness gamma endpoint.

We made this bug fix because a customer discovered with plain webpack 5 and no polyfills they would run into runtime errors due to process not being defined.

Reply:
I don't think it's specific to Next.js, it's just named that way because of how we were using it in a Next.js app.  We could change it to something like `PUBLIC_STREAMING_API_URL`?

Reply:
Tbh a bit unclear why we need this at all if `process.env.NEXT_PUBLIC_STREAMING_API_URL` resolving to `undefined` doesn't cause failures

Reply:
@calebpollman Ah it's because when `process` is undefined which is the case where you do not have a polyfill then there are errors about process being undefined and we are trying to access the object `env` on it.

Ideally this value should not affect customers at all and should only be used for debugging. 

@reesscot I thought for next web env variables they need to be prefixed with `NEXT_`? - https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser